.env.sample

```diff
@@ -1,33 +1,34 @@
 ############################### DEFAULT #####################################
 # Chain ID
-CHAIN_ID=167003
+CHAIN_ID=167004
 
 # Exposed ports
 PORT_L2_EXECTION_ENGINE_HTTP=8545
 PORT_L2_EXECTION_ENGINE_WS=8546
 PORT_L2_EXECTION_ENGINE_METRICS=6060
 PORT_L2_EXECTION_ENGINE_P2P=30303
+PORT_ZKEVM_CHAIN_PROVER_RPCD=9000
 PORT_PROMETHEUS=9090
 PORT_GRAFANA=3000
 
 # Comma separated L2 execution engine bootnode URLs for P2P discovery bootstrap
-BOOT_NODES=enode://79d73ab559debe7c420c0f97551bb8f98c37dded1ec6d3e3bca4f60bfdbbc25b62e322fcd838db381a91fe477a08755681f9ea9dd4fa49158109973fd3ac7ff7@34.71.225.47:30303,enode://9d37fd5523c0a53092b33765617753d6ca01f0c03b2f70c795bb9b38ee4ac291a7721118602e84f00984aee33a08d8bbb32398acaccdbfbaeb5925698cae2673@34.68.165.197:30303,enode://cc81cc87019b75d4e2c55d52899d57de422b4ad3e2981c789855e46b3658023d7f5eaf9343921077fbc0bc42a2f2ba464c0bb115307bc44ba4d48b5a46799f11@104.198.208.154:30303
-
-# RPC endpoints
-L1_ENDPOINT_WS=wss://l1ws.a1.taiko.xyz
+BOOT_NODES=enode://af5c8bf434ad71c1713a30428f0d643be2639f550444a9630d3ce0980c0a68cdcc2a53146448021e451adc067fe50578b4955784adce25939d06ddb142954390@35.202.212.244:30303,enode://293ddcba31a117fad992b6be0ff01594e53cb2e89d85127c63a10019edd68a007e2a074209da7917537382c8157bb06e654e8cece855df622afdfd1fba0eb65d@34.71.225.47:30303,enode://91ebd6a0355582ffc224ac774f336a5d14b67dbca42559237995701dd0ec10b473a4d3a09914ff164743ae920a42395faac7ac2b2b057c237388fde2c88957ae@35.202.142.162:30303
 
 # Taiko protocol contract addresses
-TAIKO_L1_ADDRESS=0x7B3AF414448ba906f02a1CA307C56c4ADFF27ce7
+TAIKO_L1_ADDRESS=0xAC9251ee97Ed8beF31706354310C6b020C35d87b
 TAIKO_L2_ADDRESS=0x0000777700000000000000000000000000000001
 
 # A L2 account private key for building throw-away L2 blocks, for more detailed information, please
 # see whitepaper's `5.5.1 Invalid Blocks`.
 L2_THROWAWAY_BLOCK_BUILDER_PRIVATE_KEY=92954368afd3caa1f3ce3ead0069c1af414054aefe1ef9aeacc1bf426222ce38 # LibAnchorSignature.K_GOLDEN_TOUCH_PRIVATEKEY
 
+############################### REQUIRED #####################################
+# L1 Sepolia RPC endpoints
+L1_ENDPOINT_HTTP=
```

Comment:
I'm OK with pre-set endpoints, hmm but i'm not sure if there is any open Sepolia websocket endpoint?

Reply:
yeah we can do `https://rpc.sepolia.dev`

Reply:
oh.. websocket i'm not sure. is a websocket endpoint necessary? how will they fill this out anyways, it doesn't seem clear to the user. do you know if there is some recommended place to get a websocket for sepolia like alchemy or something?

Reply:
Is it needed? Can simple-taiko-node rely on only https or it needs to have the web socket as well?

Reply:
<img width="888" alt="image" src="https://user-images.githubusercontent.com/104078303/226500037-01d3122a-e1a7-47bb-82d0-92267ca5e203.png">
it is needed, since we use event subscription a lot in client, maybe in future we can support mocking the subscription by http polls (but may also have a more serious rate-limit problems) instead of ws connection when only http(s) endpoint is provided, but for now, it is needed.